tests/integration/src/tests/smoke.rs

```diff
@@ -0,0 +1,34 @@
+#[allow(unused_imports)]
+use fluvio_command::CommandExt;
+
+#[fluvio_test_macro::fluvio_test(topic = "smoke")]
+fn smoke_test() {
+    // Magic variable
+    println!("TestOption: {:?}", &test_option);
+
+    // TODO: Create the topic. Should we move this in the generator?
+    println!("Creating the topic");
+    let mut command = get_fluvio().expect("fluvio not founded");
+    command
+        .arg("topic")
+        .arg("create")
+        .arg(&format!("{}{}", test_option.topic_name, 0)) // Fix this assumption in the smoke test
+        .arg("--replication")
+        .arg(&test_option.replication().to_string());
+    if let Some(log) = &test_option.client_log {
+        command.env("RUST_LOG", log);
+    }
+
+    let _output = command
+        .result()
+        .expect("fluvio topic create should succeed");
```

Comment:
(I know you're probably still working on this so if you already have plans for this just ignore me)

Once you have the topic-creation-via-macro all worked out, you can get rid of this shelling-out-to-create-topic code.